integration/test_redis_specific.py

```diff
@@ -25,46 +27,69 @@ def redis():
     return get_redis_client()
 
 def test_redis_driver_write(redis):
-    driver = RedisDriver(redis)
-    controller = build_flow([
-        SyncEmitSource(),
-        NoSqlTarget(Table('test', driver)),
-        Complete()
-    ]).run()
-    controller.emit({'col1': 0}, 'key').await_result()
-    controller.terminate()
-    controller.await_termination()
+    try:
+        table_name = 'test_redis_driver_write'
 
-    data = driver.redis.hgetall("storey:test/key:static")
-    data_strings = {}
-    for key, val in data.items():
-        if isinstance(key,bytes):
-            data_strings[key.decode('utf-8')] = val.decode('utf-8')
-        else:
-            data_strings[key] = val
+        driver = RedisDriver(redis)
+        controller = build_flow([
+            SyncEmitSource(),
+            NoSqlTarget(Table(table_name, driver)),
+            Complete()
+        ]).run()
+        controller.emit({'col1': 0}, 'key').await_result()
```

Comment:
I know it was there before, but why are we doing this? Await termination should be enough. If it's not, that needs to be fixed in the implementation.